README.md

```diff
@@ -100,7 +115,49 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name
+
+def initialize name
+@name = name
+end
+
+def greeter(name)
+puts "Rorrr.... I am #{@name}, I am the king of the pride "
+end
+
+end
+
+p pumba = Animal.new("pumba")
+
+
+
+class Lion < Animal
+  #sorry for not following instructions a group of lions is called a Pride, not a pack
+@@pride = []
+
+def initialize name
+@king = false
+@name = name
+if name == "simba"
+@king = true
+ @@pride << name
```

Comment:
:+1: Will give you this one, but ideally we would push an entire reference to a Lion to `@@pack`. So something like this...

```rb
def initialize
  @@pride << self  # where self = the instance being created
end
```